schematic_api/api/routes.py

```diff
@@ -608,8 +625,49 @@ def get_viz_tangled_tree_layers(schema_url, figure_type):
 
     return layers[0]
 
+def download_manifest(input_token, manifest_id, new_manifest_name='', as_json=True):
+    """
+    Download a manifest based on a given manifest id. 
+    Args:
+        input_token: token of asset store
+        manifest_syn_id: syn id of a manifest
+        newManifestName: new name of a manifest that gets downloaded.
+        as_json: boolean; If true, return a manifest as a json. Default to True
+    Return: 
+        file path of the downloaded manifest
+    """
+    # call config_handler()
+    config_handler()
+
+    # use Synapse Storage
+    store = SynapseStorage(input_token=input_token)
+    # try logging in to asset store
+    try:
+        syn = store.login(input_token=input_token)
+    except SynapseAuthenticationError as e:
+        logger.error(f'Your credentials are not valid. Please provide a valid synapse token')
+        raise e
+    except SynapseTimeoutError as e:
+        logger.error("Time out waiting for synapse to respond")
+        raise e
+    except SynapseHTTPError as e:
+        logger.error(f"A Synapse HTTP error occurred. Please see the error status:{e.response.status_code}")
```

Comment:
Here I updated the try and except syntax. But when I intentionally put in a fake token, it still only showed me the error raised by synapse python client. I guess that's fine since synapse python client is also showing informative error message. But in case I want to change it, is there a way to update the syntax?